apps/api/src/app/events/usecases/add-job/add-job.usecase.ts

```diff
@@ -27,14 +30,17 @@ export class AddJob {
       return;
     }
 
-    const digestAmount = job.type === StepTypeEnum.DIGEST ? await this.addDigestJob.execute(command) : null;
-    const delayAmount = job.type === StepTypeEnum.DELAY ? await this.addDelayJob.execute(command) : null;
+    const digestAmount =
+      job.type === StepTypeEnum.DIGEST
+        ? await this.addDigestJob.execute(AddDigestJobCommand.create({ job }))
+        : undefined;
+    const delayAmount = job.type === StepTypeEnum.DELAY ? await this.addDelayJob.execute(command) : undefined;
 
     if (job.type === StepTypeEnum.DIGEST && digestAmount === undefined) {
       return;
     }
 
-    if (digestAmount === undefined && delayAmount == undefined) {
+    if (digestAmount === undefined && delayAmount === undefined) {
```

Comment:
This was simplified based on we are returning now `undefined` in the negate brace of the ternaries of calculating `digestAmount` and `delayAmount`.